.werft/ide-integration-tests-startup.sh

```diff
@@ -169,7 +184,7 @@ JETBRAINS_TESTS="$IDE_TESTS_DIR/jetbrains"
 VSCODE_TESTS="$IDE_TESTS_DIR/vscode"
 SSH_TESTS="$IDE_TESTS_DIR/ssh"
 
-IDE_TEST_LIST=("$SSH_TESTS $VSCODE_TESTS $JETBRAINS_TESTS")
+IDE_TEST_LIST=("$SSH_TESTS" "$VSCODE_TESTS" "$JETBRAINS_TESTS")
```

Comment:
This fixes a bug in the IDE integration tests which would cause them to fail with `cd: /workspace/test/tests/ide/ssh /workspace/test/tests/ide/vscode /workspace/test/tests/ide/jetbrains: No such file or directory` because it got interpreted as a single string before. Now it's three separate strings ☺️